client/my-sites/domains/domain-management/list/empty-state/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import page from '@automattic/calypso-router';
+import { __experimentalText as Text, Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import './style.scss';
+
+export default function EmptyState() {
+	const translate = useTranslate();
+
+	const handleClickUseMyDomain = () => {
+		recordTracksEvent( 'calypso_domain_management_list_empty_state_use_my_domain_click' );
+		page( '/setup/domain-transfer' );
```

Comment:
I think we can simply add this as a `href` to the `<Button />` below; it will just work without manual page routing, what do you think? (The Tracks event will also still work)

Reply:
Sure! Updated 🙂